docs/templates/release-notes-regular.md

```diff
@@ -0,0 +1,11 @@
+This release includes various new features, stability improvements, and security fixes. If you are on a Mac or PC you should also update Docker Desktop for security reasons.
```

Comment:
While reading this sentence, I think that we can improve the wording a bit for subsequent releases:

```suggestion
This release includes various new features, stability improvements, and security fixes **(adjust accordingly)**. If you are on a Mac or PC please also update Docker Desktop to the latest version to get the latest security fixes
```